doc/manual/src/glossary.md

```diff
@@ -115,14 +115,14 @@
     then be built.
 
   - [reference]{#gloss-reference}\
-    A store path `P` is said to have a reference to a store path `Q` if
-    the store object at `P` contains the path `Q` somewhere. The
-    *references* of a store path are the set of store paths to which it
-    has a reference.
+    A [store object] `O` is said to have a *reference* to a store object `P` if a [store path] to `P` appears in the contents of `O`.
+    The *references* of a store object `O` are the set of store objects to which `O` has a reference.
```

Comment:
I sort of just want to drop this sentence. Refernces = set of references themselves is arguably better English than set of things referred to. But it is nice to still not talk about store paths as that is just one means of referencing.

The references of a store object are a lot like the string contexts of a string.

Reply:
The references of a store object are a lot like the string contexts of a string. I might also make sense to call them both references (string references, store object references; string contex, store object context). The glossary for reference could refer to both kinds of references.

Reply:
```suggestion
```

With pleasure. I also found it weird and barely helpful, but didn't want to change too much.